platform/fabric/core/generic/channel.go

```diff
@@ -172,6 +177,27 @@ func newChannel(network *network, name string, quiet bool) (*channel, error) {
 	return c, nil
 }
 
+var defaultCacheSize = 100
+
+// cacheSizeFromConfig returns the KVS cache size from current configuration.
+// Returns defaultCacheSize, if no configuration found.
+// Returns an error and defaultCacheSize, if the loaded value from configuration is invalid (must be >= 0).
+func cacheSizeFromConfig(sp view2.ServiceProvider) (int, error) {
+	configService := view2.GetConfigService(sp)
+	configKey := "fabric.vault.txidstore.cache.size"
```

Comment:
You could probably use this as a constant.

I know it's implemented like this also for the others, but maybe it would make sense for the config service to provide methods like
```
GetInt(key string) (int, boolean)
```

Reply:
> You could probably use this as a constant.

Good catch!  added constants for the config keys used in the KVS.

Regarding your other proposal to change the config API, I think this is a great idea. We should revisit the configuration component and do a cleanup in the entire repo. @alexandrosfilios could you please create an issue for that?

For this PR I would like to keep the `GetInt` methods as is to follow the existing pattern. I've also seen that even that change required a few tweaks here and there to make other unit tests run again. I've added a few todos to address that.